name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/preferences/KrakenPreferencePage.java

```diff
@@ -0,0 +1,52 @@
+package name.abuchen.portfolio.ui.preferences;
+
+import org.eclipse.jface.preference.BooleanFieldEditor;
+import org.eclipse.jface.preference.FieldEditorPreferencePage;
+import org.eclipse.jface.preference.StringFieldEditor;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.widgets.Link;
+
+import name.abuchen.portfolio.ui.Messages;
+import name.abuchen.portfolio.ui.UIConstants;
+import name.abuchen.portfolio.ui.util.DesktopAPI;
+
+public class KrakenPreferencePage extends FieldEditorPreferencePage
+{
+
+    public KrakenPreferencePage()
+    {
+        super(GRID);
+
+        setTitle(Messages.PrefTitleKraken);
+        setDescription(Messages.PrefDescriptionKraken);
+    }
+
+    public void createFieldEditors()
+    {
+        final String krakenApiSettingsUrl = "https://www.kraken.com/u/settings/api";
+        Link link = new Link(getFieldEditorParent(), SWT.NONE);
+        link.setText("<a>" + krakenApiSettingsUrl + "</a>"); //$NON-NLS-1$
+        link.setLayoutData(new GridData(SWT.FILL, SWT.TOP, false, false, 3, 1));
+        link.addSelectionListener(new SelectionAdapter()
+        {
+            @Override
+            public void widgetSelected(final SelectionEvent event)
+            {
+                DesktopAPI.browse(krakenApiSettingsUrl); // $NON-NLS-1$
+            }
+        });
+
+        addField(new StringFieldEditor(UIConstants.Preferences.KRAKEN_API_KEY, Messages.PrefKrakenAPIKey,
```

Comment:
Looks like the preference keys are not needed for the QuoteProvider implementation. If so, then I would prefer to have them in the next PR.